### PR TITLE
Add more detail to H.264 and GIF commands

### PR DESCRIPTION
**Pull request by kfrn**
I hope I'm not stepping on any toes, but I thought it would be useful to add a little more detail. With the gif command (contributed by @retokromer, I think? Thank you!), it was to understand it better for myself more than anything - my text could be improved I think, so feel free to edit.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -153,9 +153,12 @@
...
           <p><code>ffmpeg -i <i>input_file</i> -c:v libx264 -pix_fmt yuv420p -preset veryslow -crf 18 -c:a copy <i>output_file</i></code></p>
           <dl>
-            <dt>-preset <i>veryslow</i></dt><dd>This option tells ffmpeg to use the slowest preset possible for the best compression quality.</dd>
-            <dt>-crf <i>18</i></dt><dd>Specifying a lower CRF will make a larger file with better visual quality. 18 is often considered a “visually lossless” compression.</dd>
+            <dt>-preset <i>veryslow</i></dt><dd>This option tells ffmpeg to use the slowest preset possible for the best compression quality.<br>
+            Available presets, from slowest to fastest, are: <code>veryslow</code>, <code>slower</code>, <code>slow</code>, <code>medium</code>, <code>fast</code>, <code>faster</code>, <code>veryfast</code>, <code>superfast</code>, <code>ultrafast</code>.</dd>
+            <dt>-crf <i>18</i></dt><dd>Specifying a lower CRF will make a larger file with better visual quality. The scale ranges between 0-51, with 0 being lossless and 51 the worst possible quality.<br>
```
Comment (retokromer) [inline index.html]: [As long as I know](https://avpres.net/FFmpeg/im_H264.html) the interval [0..51] is valid only for 8-bit, but for 10-bit it’s [0..63].

File: index.html [hunk 2]
```diff
@@ -819,7 +819,7 @@
...
             <dt>-filter_complex "fps=<i>frame rate</i>,scale=<i>width</i>:<i>height</i>,palettegen"</dt><dd>a complex filtergraph.<br>
               Firstly, the fps filter sets the frame rate.<br>
-              Then the scale filter resizes the image. You can specify both the width and the height, or specify a value for one and use a scale value of <i>-1</i> for the other to preserve the aspect ratio. (For example, <code>500:-1</code> would create a GIF 500 pixels wide and with a height proportional to the original video). In the first script above, <code>:flags:lanczos</code> specifies that the Lanczos rescaling algorithm will be used to resize the image.<br>
+              Then the scale filter resizes the image. You can specify both the width and the height, or specify a value for one and use a scale value of <i>-1</i> for the other to preserve the aspect ratio. (For example, <code>500:-1</code> would create a gif 500 pixels wide and with a height proportional to the original video). In the first script above, <code>:flags=lanczos</code> specifies that the Lanczos rescaling algorithm will be used to resize the image.<br>
```
Comment (retokromer) [inline index.html]: Did we not use capitals for the abbreviation, like TIFF, DCP, GIF, etc.? (No, GIF it’s not me :-)

File: index.html (hunk 1 above)
Comment (kfrn) [reply index.html]: Ah, I had just looked to the [wiki](https://trac.ffmpeg.org/wiki/Encode/H.264). Thanks for the link. Perhaps I should add a note explaining that the 0-51 scales applies for `yuv420p` only.

File: index.html (hunk 2 above)
Comment (kfrn) [reply index.html]: Will change!
The real question ... do you pronounce it with a hard or soft 'g'? ;)

**Review by retokromer — APPROVED** [review]

File: index.html (hunk 2 above)
Comment (retokromer) [reply index.html]: With a hard, but perhaps this is not English…

**Comment by retokromer** [comment]
Thank you!

File: index.html (hunk 2 above)
Comment (kfrn) [reply index.html]: It's a matter of [great debate](https://en.wikipedia.org/wiki/GIF#Pronunciation_of_GIF), you see! I have always said it with a hard 'g'.

**Comment by kfrn** [comment]
As always, thank you for your careful revisions Reto!

**Comment by ablwr** [comment]
Haha I love when these things happen while I snooze!! Thank you so much @kfrn! I had felt the same about the GIF one... and also considered breaking it up officially into two commands. Thanks for these changes!

**Pull request merged**